packages/strapi-admin/admin/src/containers/Roles/EditPage/index.js

```diff
@@ -73,7 +73,7 @@ const EditPage = () => {
         body: data,
       });
 
-      if (role.code !== 'strapi-super-admin' && !isEmpty(permissionsToSend)) {
+      if (role.code !== 'strapi-super-admin') {
```

Comment:
Does it mean that we will call permissions even if permissionsToSend is empty?

Reply:
Yes it's for the PUT a role can have zero permission